Realm/RLMObject_Private.h

```diff
@@ -61,6 +61,8 @@ FOUNDATION_EXTERN BOOL RLMObjectBaseAreEqual(RLMObjectBase *o1, RLMObjectBase *o
 // Get ObjectUil class for objc or swift
 FOUNDATION_EXTERN Class RLMObjectUtilClass(BOOL isSwift);
 
+FOUNDATION_EXTERN const NSUInteger RLMDescriptionMaxDepth;
```

Comment:
Should just be a `static const` variable rather than an `extern` one. The point of externing string constants is to ensure that they can be compared with pointer equality.